fastscape/tests/test_processes_initial.py

```diff
@@ -0,0 +1,64 @@
+import numpy as np
+
+import pytest
+
+from fastscape.processes import (BareRockSurface,
+                                 Escarpment,
+                                 FlatSurface,
+                                 NoErosionHistory)
+
+def test_flat_surface():
+    np.random.seed(5)
+
+    grid = (3, 2)
+    elevation = np.random.rand(*grid)
+
+    np.random.seed(5)
+    p = FlatSurface(shape=grid)
+    p.initialize()
+    
+    np.testing.assert_equal(grid, p.shape)
+    np.testing.assert_equal(elevation, p.elevation)
+
+
+@pytest.mark.parametrize('inputs, expected_result', [
+    ({'x_left': 2,
+      'x_right': 3,
+      'elevation_left': 7.,
+      'elevation_right': 2.,
+      'shape': np.array([3,2]),
+      'x': np.array([1., 11., 21., 31.])},  np.array([[7., 2.],[7., 2.],[7., 2.]])),
+    ({'x_left': 2,
+      'x_right': 3,
+      'elevation_left': 2.,
+      'elevation_right': 7.,
+      'shape': np.array([3,2]),
+      'x': np.array([1., 11., 21., 31.])},  np.array([[2., 7.],[2., 7.],[2., 7.]])),
+    ({'x_left': 2,
```

Comment:
It'd be more useful to test the case where `x_left` and `x_right` have the same value but where plateau elevations have different values, because of the `if scarp_width > 0` in the function tested.